CHANGELOG.md

```diff
@@ -4,6 +4,19 @@
 * Enhancement - Add clear info message when chunks are removed because `storage.total_limit_size` is reached [fluent-bit:6719](https://github.com/fluent/fluent-bit/pull/6719)
 
 
+### 2.31.8
+This release includes:
+* Fluent Bit [1.9.10](https://fluentbit.io/announcements/v1.9.10/)
+* Amazon CloudWatch Logs for Fluent Bit 1.9.2
+* Amazon Kinesis Streams for Fluent Bit 1.10.1
+* Amazon Kinesis Firehose for Fluent Bit 1.7.1
+
+Compared to `2.31.7` this release adds:
```

Comment:
@Claych since I merged the init gov cloud fix it also is going out in this release too I think. Needs to be called out or reverted. 

We're not supposed to add features in CVE releases, so technically we should revert it. Just in case my change introduced a bug which would block folks from consuming the CVE fixes. 



https://github.com/aws/aws-for-fluent-bit/commits/mainline